package exception;

public class demo {
	public static void main(String[]args)
	{
		System.out.println("the exception we got is :");
		division.divi();
	}

}
